lib/datadog/appsec/event.rb

```diff
@@ -137,6 +137,24 @@ def build_service_entry_tags(event_group)
         end
         # rubocop:enable Metrics/MethodLength
 
+        def add_event_tags(scope, waf_result)
+          if scope.service_entry_span
+            scope.service_entry_span.set_tag('appsec.blocked', 'true') if waf_result.actions.include?('block')
+            scope.service_entry_span.set_tag('appsec.event', 'true')
+          end
+
+          # Propagate to downstream services the information that the current distributed trace is
+          # containing at least one ASM security event
```

Comment:
I would put this comment before `scope.trace.keep!`, it's more relevant there

Reply:
Fixed!